test/tld.js

```diff
@@ -291,7 +291,7 @@ describe('tld.js', function () {
     });
 
     it('should return the initial value if it is not a valid hostname', function(){
-      expect(tld.extractHostname(42)).to.equal('42');
+      expect(tld.extractHostname(42)).to.equal(42);
```

Comment:
As mentioned above - "return the initial value" feels like it should return an integer, not a string. But this is a change in behavior.